_docs/pages/supported-tracers.md

```diff
@@ -6,30 +6,12 @@
 
 ## Jaeger
 
-[Jaeger \ˈyā-gər\](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
+[Jaeger \ˈyā-gər\ˈ](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
```

Comment:
ah -- I was fixing the fact that there was an orphan `\` character by guessing what was supposed to be escaped :)  Removed.